modules/fieldlinessequence/rendering/renderablefieldlinessequence.cpp

```diff
@@ -1076,48 +948,54 @@ void RenderableFieldlinesSequence::render(const RenderData& data, RendererTasks&
         _shaderProgram->setUniform("modelViewProjection",
```

Comment:
Can't comment earlier, but you can decrease the overall indentation of this function by checking for `if (_activeTriggerTimeIndex == -1)` and have an early `return`.  That way the rest of the function can be reduced by a level